src/function/table/system/duckdb_constraints.cpp

```diff
@@ -171,7 +172,13 @@ void DuckDBConstraintsFunction(ClientContext &context, const FunctionData *bind_
 				column_index_list.push_back(bound_not_null.index);
 				break;
 			}
-			case ConstraintType::FOREIGN_KEY:
+			case ConstraintType::FOREIGN_KEY: {
+				auto &bound_foreign_key = (BoundForeignKeyConstraint &)bound_constraint;
```

Comment:
This does not appear to be covered, could you add a foreign key constraint to the test that runs `duckdb_constraints`?